common/persistence/src/main/java/app/coronawarn/server/common/persistence/service/DiagnosisKeyService.java

```diff
@@ -113,9 +151,13 @@ public void applyRetentionPolicy(int daysToRetain) {
         .ofInstant(Instant.now(), UTC)
         .minusDays(daysToRetain)
         .toEpochSecond(UTC) / SECONDS_PER_HOUR;
-    int numberOfDeletions = keyRepository.countOlderThanOrEqual(threshold);
+    int numberOfDeletions = keyRepository.countOlderThanOrEqual(threshold, countryCode);
     logger.info("Deleting {} diagnosis key(s) with a submission timestamp older than {} day(s) ago.",
         numberOfDeletions, daysToRetain);
```

Comment:
why does the logger not record the countryCode?
Can the countryCode be reliably reconstructed by looking at log infos before/after this log line?

Reply:
Good point. Ideally we should enhance all logs with a country prefix.